Mathlib/Order/ConditionallyCompleteLattice/Basic.lean

```diff
@@ -61,6 +61,26 @@ noncomputable instance {α : Type _} [SupSet α] : SupSet (WithBot α) :=
 noncomputable instance {α : Type _} [Preorder α] [InfSet α] : InfSet (WithBot α) :=
   ⟨(@instSupSetWithTop αᵒᵈ _).supₛ⟩
 
+theorem WithTop.supₛ_eq [Preorder α] [SupSet α] {s : Set (WithTop α)} (hs : ⊤ ∉ s)
+    (hs' : BddAbove ((↑) ⁻¹' s : Set α)) : supₛ s = ↑(supₛ ((↑) ⁻¹' s) : α) :=
+  (if_neg hs).trans $ if_pos hs'
```

Comment:
mathport seems to declare that `<|` is more idiomatic than `$`; probably best in future just to leave it.